runner/database.go

```diff
@@ -483,7 +483,7 @@ func (ec EvalContext) EvalDatabasePanel(
 			return err
 		}
 
-		if vendor == "sqlite3_extended" {
+		if vendor == "sqlite3_extended" && !ec.settings.CacheMode {
```

Comment:
Why should we avoid setting these pragmas in cache mode? I think they need to be set for every connection.

Reply:
Oh I first thought they only need to be set once.